ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py

```diff
@@ -3608,6 +3606,67 @@ def test_torch_instance_cumsum(
     )
 
 
+@st.composite
+def _get_clamp_inputs(draw):
+    shape = draw(
+        helpers.get_shape()
+    )
+    x_dtype, x = draw(
+        helpers.dtype_and_values(
+            available_dtypes=helpers.get_dtypes("float"),
+            min_value=-1e04,
+            max_value=1e04,
+            allow_inf=False,
+            shape=shape,
+        )
+    )
+    min = draw(
+        helpers.array_values(dtype=x_dtype[0], shape=shape, min_value=-100, max_value=0)
+    )
+    max = draw(
+        helpers.array_values(dtype=x_dtype[0], shape=shape, min_value=1, max_value=100)
+    )
+    return x_dtype, x, min, max
+
+
+# clamp
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="torch.tensor",
+    method_name="clamp",
+    dtype_and_x_min_max=_get_clamp_inputs().filter(lambda x: "float16" not in x[0])
```

Comment:
We shouldn't filter the data types generated from the helper, they won't be generated if the method does not support it, we should flag it using the `unsupported` decorator.